api/RTCRtpEncodingParameters.json

```diff
@@ -239,8 +239,8 @@
           },
           "status": {
             "experimental": false,
-            "standard_track": false,
-            "deprecated": true
+            "standard_track": true,
+            "deprecated": false
```

Comment:
This is in the spec, and implemented everywhere. There is no evidence that it is deprecated.